sw/host/cryptotest/testvectors/parsers/cryptotest_util_test.py

```diff
@@ -0,0 +1,220 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import tempfile
+import os
+import unittest
+from typing import List, Dict
+
+from cryptotest_util import parse_rsp
+
+
+def createRsp(content: bytes) -> str:
+    with tempfile.NamedTemporaryFile(delete=False, dir=os.environ["TEST_TMPDIR"]) as temp:
+        temp.write(content)
+        return temp.name
+
+
+def checkResult(t: unittest.TestCase, expected: List[Dict[str, str]], actual: List[Dict[str, str]]):
```

Comment:
Much simpler. thanks!

Reply:
Ah I was thinking we can probably just omit this `checkResult` function entirely and call `t.assertEqual` directly on the two lists in the test case without checking anything else. I don't think there's a need to check the equality of the dictionaries explicitly since Python defines equality of lists as equality of all the corresponding elements and the equality check already handles the length checking.

Reply:
I was concerned about rejecting an implementation that returned the correct test vectors in the wrong order, but that really can't happen. I think we can simplify this even more.

Reply:
Fixed